interactive_engine/compiler/src/main/java/com/alibaba/graphscope/common/jna/type/ResultOpt.java

```diff
@@ -20,7 +20,8 @@
 
 public enum ResultOpt implements IntEnum<ResultOpt> {
     EndV,
-    AllV;
+    AllV,
+    AllVe;
```

Comment:
`AllVe` is strange, either `AllVE`, if the compiler complains, change everything to full name, ie., `EndVertex`, `AllVertex`, `AllVertexEdge`